src/polyswarm_api/__init__.py

```diff
@@ -586,14 +614,14 @@ def _fix_result(self, result):
 
         return {'file_path': out_path, 'status': 'OK', 'file_hash': h}
 
-    async def download_files(self, hashes, destination_dir, with_metadata=True, hash_type='sha256'):
+    async def download_files(self, hashes, destination_dir, with_metadata=True, hash_type=None):
         """
         Download files  via the PS API
 
         :param hashes: Hashes of the files you wish to download
         :param destination_dir: Directory you wish to save the files in
         :param with_metadata: Whether to save related file metadata into an associated JSON files
-        :param hash_type: Hash type [sha256|sha1|md5]
+        :param hash_type: Hash type [default:sha256|sha1|md5]
```

Comment:
autodetect. update the other docstrings too

Reply:
done